_posts/2016-01-13-state-of-the-lab.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: State of the Lab 2016
+author: whoburg
+---
+
+A recent e-mail I sent to the lab:
+
+<tt>
+> Folks:
+
+> Welcome back from the holidays and happy 2016. I thought last night's State of the Union address a good occasion to reflect on our first 1.4 years of existence and where we're headed.
```

Comment:
Good question.

I could be wrong, but I think this usage is correct. I'll argue for why, but please someone tell me if I'm wrong. I'm basing my opinion here on intuition only and have no actual English training to back up my claims.

I'm using a construction similar to one that's used in Robert Frost's "Stopping by Woods on a Snowy Evening", when he says "My little horse must think it queer" -- the pattern is "think [something] [adjective]" -- note he doesn't say "horse must think it is queer" or "horse must think it was queer". So, mine is "thought [last night's state of the union address] [a good occasion]". The only difference is that "a good occasion" is a noun phrase, not an adjective. But it seems similar to me.

How's that for a response you probably didn't expect to your innocent question?